src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/DefaultFeedbackFileLoggerProviderFactory.cs

```diff
@@ -30,13 +30,7 @@ public override object GetOrCreate()
         {
             lock (_creationLock)
             {
-                if (_currentFileLogWriter != null)
```

Comment:
@NTaylorMullen I removed this guard to enable using the same log file between RZLS/O# & VS HTML CSharp Language Server. I was hoping to have the logs interleaved to simplify back-tracing investigations. Please let me know if you believe this may be problematic.

Reply:
Hmm I do think this will be problematic. If you close/open solutions we'll never create new log files meaning we wont know what's the "current" session. Because of the asynchronous nature of the two language servers (Razor's + HTML/C#) it would probably make sense to have two separate log files to be totally honest. Maybe it makes sense for the GetORCreate to take an identifier and for us to dispose the previous log factory if it existed already for the provided identifier? To make it even nicer we could even further expose a specific HTMlC# factory and a Razor factory so that the idea of providign an identifier is abstracted away

Reply:
Thanks, updated accordingly (2 separate log files for Razor & HTML/C# Language servers).

https://github.com/dotnet/aspnetcore-tooling/pull/2936/commits/cf1c117514d53f5012a60a7f1f9e04f94de03f45

Reply:
Left another comment but which piece ensures they're totally different log files?